openmls/src/treesync/mod.rs

```diff
@@ -347,21 +345,21 @@ impl fmt::Display for RatchetTree {
     }
 }
 
-/// The [`TreeSync`] struct holds an [`MlsBinaryTree`] instance, which contains
+/// The [`TreeSync`] struct holds an `MlsBinaryTree` instance, which contains
 /// the state that is synced across the group, as well as the [`LeafNodeIndex`]
 /// pointing to the leaf of this group member and the current hash of the tree.
 ///
 /// It follows the same pattern of tree and diff as the underlying
-/// [`MlsBinaryTree`], where the [`TreeSync`] instance is immutable safe for
-/// merging a [`TreeSyncDiff`], which can be created, staged and merged (see
-/// [`TreeSyncDiff`]).
+/// `MlsBinaryTree`, where the [`TreeSync`] instance is immutable safe for
+/// merging a `TreeSyncDiff`, which can be created, staged and merged (see
+/// `TreeSyncDiff`).
```

Comment:
If `TreeSync` is becoming `pub`, we should update the docs to something that makes sense to the application developer.